e2e/tests/user/registration-ui.spec.ts

```diff
@@ -50,19 +50,6 @@ describe('User registration tests:', () => {
     // uncheck a checkbox then check NEXT button is again disabled
     await (await createAccountPage.getTermsOfUseCheckbox()).unCheck();
     expect(await nextButton.isCursorNotAllowed()).toEqual(true);
-  });
-
-
-  test('Loading User information page', async () => {
```

Comment:
Not really gain anything by having two tests. Combine two into one in order to run it quicker.